lint/rule_registry.go

```diff
@@ -18,12 +18,12 @@ import (
 	"fmt"
 )
 
-// Rules is a registry for registering and looking up rules.
-type Rules map[RuleName]Rule
+// RuleRegistry is a registry for registering and looking up rules.
+type RuleRegistry map[RuleName]ProtoRule
```

Comment:
Consider change it to `ProtoRuleRegistry` as well or `Register` to `RegisterProtoRule`.

Reply:
I think `Register` to `RegisterProtoRules` is better, considering how you use `RuleRegistry` in the `AddRules` function in each rule package.